service.nextup.notification/service.py

```diff
@@ -34,16 +37,32 @@ def logMsg(self, msg, lvl=1):
     def ServiceEntryPoint(self):
         player = Player()
         monitor = xbmc.Monitor()
-
+        
         lastFile = None
         lastUnwatchedFile = None
+
+        ga = GoogleAnalytics()
```

Comment:
if you want to make it opt-in you would need to detail every piece of data that gets uploaded in a manner that's fully understandable for every user.

personally, i would stay away as far as possible from this.